dataplicity/m2mmanager.py

```diff
@@ -271,8 +271,9 @@ def on_instruction(self, sender, data):
             self.open_echo_service(port)
         elif action == 'open-portforward':
             service = data['service']
+            local_port = int(data['port'])
```

Comment:
I am also concidering regular `get()` call here (as in data.get('port', None) ) and only assert that the port is set in the final `if branch` (as in - line 322 onwards in file dataplicity/portforward.py)